fix gpu example

Borda opened the pull request:
## What does this PR do?

Fixes https://github.com/PyTorchLightning/pytorch-lightning/pull/2285#issuecomment-652990997

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

awaelchli commented:
@Borda the gpu template and cpu template were the same and both were running on cpu. Now I changed it. is it ok?

Borda commented:
> Borda the gpu template and cpu template were the same and both were running on cpu. Now I changed it. is it ok?

I have added ` --gpus 1` for GPU example, that is fine, right?

awaelchli commented:
@Borda yes. I would maybe test that in that case the model is actually on gpu, so we know the arg is correctly passed to the template .)
